api/tsconfig.json

```diff
@@ -13,5 +13,5 @@
 		"declaration": true,
 		"resolveJsonModule": true
 	},
-	"exclude": ["node_modules", "dist"]
+	"exclude": ["node_modules", "dist", "extensions"]
```

Comment:
Oh, I was actually using some extensions written with typescript and placed in the default folder `api/extensions`. Since they weren't written very rigorously, they were raising multiple errors on build, and I decided to just exclude them from build, and forgot about that change.

Reply:
I don't see any difference with it not being there, nor have had any problems before. I'll remove it 👍🏻